qualtran/bloqs/arithmetic/comparison.py

```diff
@@ -1017,3 +1024,135 @@ def _eq_k() -> EqualsAConstant:
 
 
 _EQUALS_K_DOC = BloqDocSpec(bloq_cls=EqualsAConstant, examples=[_eq_k])
+
+
+@frozen
+class CLinearDepthGreaterThan(Bloq):
+    r"""Controlled greater than between two integers.
+
+    Implements |c>|a>|b>|t> => |a>|b>|t ⨁ ((a > b)c)> using $n+2$ Toffoli gates.
```

Comment:
nit: use Latex in docstrings?

Reply:
The RHS of the => seems to be missing the $|c\rangle$ register before $|a\rangle$

Reply:
thanks for catching this